src/Playwright.Tests/PageEvaluateTests.cs

```diff
@@ -388,18 +386,6 @@ public async Task ShouldWorkForCircularObject()
             Assert.NotNull(result);
         }
 
-        [PlaywrightTest("page-evaluate.spec.ts", "should be able to throw a tricky error")]
-        [Ignore("v1.22 blocker")]
-        public async Task ShouldBeAbleToThrowATrickyError()
```

Comment:
What's up with this one?

Reply:
The upstream test makes no sense anymore:

```js
it('should be able to throw a tricky error', async ({ page }) => {
  const windowHandle = await page.evaluateHandle(() => window);
  const errorText = await windowHandle.jsonValue().catch(e => e.message); // [0]
  console.log(errorText, typeof errorText);
  const error = await page.evaluate(errorText => {
    throw new Error(errorText);
  }, errorText).catch(e => e);
  expect(error.message).toContain(errorText);
});
```

Previously before the circular reference support, [0] was throwing because `window` had circular references. But now it's not throwing anymore, and it's just `ref: <Window>` which makes the whole test kinda unnecessary. because there are no exception messages to compare with.